site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -86,15 +72,15 @@ DevTools extension can add UI elements to the DevTools window:
 
 - A _panel_ is a top-level tab, like the Elements, Sources, and Network panels.
 - A _sidebar pane_ presents supplementary UI related to a panel. The Styles, Computed Styles, and
-  Event Listeners panes on the Elements panel are examples of sidebar panes. (Note that the
-  appearance of sidebar panes may not match the image, depending on the version of Chrome you're
-  using, and where the DevTools window is docked.)
+  Event Listeners panes on the Elements panel are examples of sidebar panes. Depending on the
+  version of Chrome you're using and where the DevTools window is docked, your sidebar panes might
+  look like the following example image:
 
 {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/TDNgfhI9byR4eeGQ0Xxv.png", class="screenshot",
     alt="DevTools window showing Elements panel and Styles sidebar pane.", height="302", width="770" %}
 
 Each panel is its own HTML file, which can include other resources (JavaScript, CSS, images, and so
```

Comment:
Please [add captions to images](https://web.dev/handbook/markup-media/#captions) when you encounter them.

You don't need to go back and fix images you may have missed in other edits. This isn't as crucial as the quality edits, and I'm sure we'll get them all eventually.

Reply:
Thanks for the heads-up!